scripts/umount.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Variables set from /etc/fobnail.cfg
+[ -n "$MNT_DIR" ] || die "Target directory not set"
+[ -n "$MAPPER_DEV" ] || die "Mapper device name not set"
+
+# Variables set by Systemd
+[ -n "$RUNTIME_DIRECTORY" ] || die "RUNTIME_DIRECTORY not set"
+
+LOOP_DEV_FILE="$RUNTIME_DIRECTORY/fobnail.loopdev"
+umount "$MNT_DIR"
+cryptsetup close "$MAPPER_DEV"
+losetup -d "$(cat "$LOOP_DEV_FILE")"
```

Comment:
This is actually correct:
```
export LOOP_DEV_FILE="path with space"
echo error > path
echo error > with
echo error > space
echo ok > 'path with space'
x="$(cat $LOOP_DEV_FILE)"
akowalski@akowalski-Opti:/tmp/test$ echo $x
# this will give: error error error
# but this
x="$(cat "$LOOP_DEV_FILE")"
echo $x
# gives: ok
# and this doesn't parse well
x="$(cat \"$LOOP_DEV_FILE\")"
cat: '"path': No such file or directory
cat: 'space"': No such file or directory
```

Reply:
@krystian-hebel confirmed working from a script file too. Works both from shell and from script.